cloudSupport/src/main/scala/cromwell/cloudSupport/gcp/gcs/GcsStorage.scala

```diff
@@ -0,0 +1,56 @@
+package cromwell.cloudSupport.gcp.gcs
+
+import com.google.api.client.googleapis.javanet.GoogleNetHttpTransport
+import com.google.api.client.googleapis.media.MediaHttpUploader
+import com.google.api.client.json.jackson2.JacksonFactory
+import com.google.api.gax.retrying.RetrySettings
+import com.google.cloud.storage.StorageOptions
+import com.google.api.services.storage.Storage
+import com.google.auth.Credentials
+import com.google.cloud.storage.contrib.nio.CloudStorageConfiguration
+import com.typesafe.config.ConfigFactory
+import cromwell.cloudSupport.gcp.GoogleConfiguration
+import cromwell.cloudSupport.gcp.http.GoogleHttpTransportOptions.TransportOptions
+import net.ceedubs.ficus.Ficus._
+
```

Comment:
This was really just moved around and has already faced an angry flock of seabirds but have at it